JobSchema.md

```diff
@@ -0,0 +1,221 @@
+Job Schema Service
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+Job Schema Service (JSS) takes a request containing data for a job posting and returns a response conforming to the [schema.org JobPosting schema](﻿http://schema.org/JobPosting).
+
+## Request Structure
```

Comment:
For below json, I would add something to indicate what are optional, what are required?

Reply:
I did. It's at the bottom.